### DISCUSSION TITLE
there is no option to get all fields coz [Oct 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4309] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **KVY**  
#### But as user you need to know how many members in your group. I mean amount. I need what I describe because of it... So what the options to get members length then?  
#  
> 🙋‍♂️ **KVY**  
#### friend backender tell me "select top (@p0) count([id]) as [count] from [my_table]" but is there ability to access the database directly in NocoDB  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1034022268029894726)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
